modules/site-health/admin-site-health.php

```diff
@@ -108,6 +108,96 @@ function ( &$value, $key ) {
+					// translators: the placeholder is the option stored in database.
+					$value = __( 'Browser prefered language activated (%s)', 'polylang' );
+				} else {
+					// translators: the placeholder is the option stored in database.
+					$value = __( 'Browser prefered language deactivated (%s)', 'polylang' );
+				}
+				break;
+			case 'rewrite':
+				if ( 1 === $value ) {
+					// translators: the placeholder is the option stored in database.
+					$value = __( 'Remove /language/ in pretty permalinks (%s)', 'polylang' );
+				} else {
+					// translators: the placeholder is the option stored in database.
+					$value = __( 'Keep /language/ in pretty permalinks (%s)', 'polylang' );
+				}
+				break;
+			case 'hide_default':
+				if ( 1 === $value ) {
+					// translators: the placeholder is the option stored in database.
+					$value = __( 'Hide URL language information for default language (%s)', 'polylang' );
+				} else {
+					// translators: the placeholder is the option stored in database.
+					$value = __( 'Display URL language information for default language (%s)', 'polylang' );
+				}
+				break;
+			case 'force_lang':
+				switch ( $value ) {
+					case '0':
+						// translators: the placeholder is the option stored in database.
+						$value = __( 'The language is set from content (%s)', 'polylang' );
+						break;
+					case '1':
+						// translators: the placeholder is the option stored in database.
+						$value = __( 'The language is set from the directory name in pretty permalinks (%s)', 'polylang' );
+						break;
+					case '2':
+						// translators: the placeholder is the option stored in database.
+						$value = __( 'The language is set from the subdomain name in pretty permalinks (%s)', 'polylang' );
+						break;
+					case '3':
+						// translators: the placeholder is the option stored in database.
+						$value = __( 'The language is set from different domains (%s)', 'polylang' );
+						break;
+				}
+				break;
+			case 'redirect_lang':
+				if ( 1 === $value ) {
+					// translators: the placeholder is the option stored in database.
+					$value = __( 'The front page URL contains the language code instead of the page name or page id (%s)', 'polylang' );
+				} else {
+					// translators: the placeholder is the option stored in database.
+					$value = __( 'The front page URL contains the page name or page id instead of the language code (%s)', 'polylang' );
+				}
+				break;
+			case 'media_support':
+				if ( true === $value ) {
+					// translators: the placeholder is the option stored in database.
+					$value = __( 'The media are translated (%s)', 'polylang' );
+				} else {
+					$value = __( 'The media are not translated (0)', 'polylang' );
+				}
+				break;
+
+			case 'sync':
+				if ( is_array( $value ) && empty( $value ) ) {
+					$value = __( 'Synchronization disabled (0)', 'polylang' );
+				}
+				break;
+		}
+		if ( ! is_array( $value ) ) {
+			$value = sprintf( $value, $db_value );
```

Comment:
As PHPStan mentioned : 
```
  Line   modules/site-health/admin-site-health.php                 
 ------ ---------------------------------------------------------- 
  196    Parameter #1 $format of function sprintf expects string,  
         bool|int|string given.                                    
  196    Parameter #2 ...$values of function sprintf expects       
         bool|float|int|string|null, array|bool|int|string given
```
May be check the value's type before using `sprintf()` on it ?

Reply:
I really don't know how to manage this.
I've change to  `if ( is_string( $value ) ) {` but I always have the same error.
I wonder if the problem isn't in my PHPDoc declaration. I'm not sure of 
```
	 * @param string|array|bool|int $value option value.
	 * @return string|array $value option value.
```
it looks like a bag where  I put all possible type :-)

Reply:
I don't really know the best way to handle this.

@Chouby What do you think ? 

Should we cast the expected type like this : 
```
     if ( ! is_array( $value ) ) {
         $value = (string) $value;
         if ( ! is_array( $db_value ) ) {
            $value = sprintf( $value, $db_value );
         }
      }
```